common.vars.pkr.hcl

```diff
@@ -1 +1 @@
-postgres-version = "15.1.0.62"
+postgres-version = "15.1.0.63-rc"
```

Comment:
if we need to test this on staging, could you create the build off this branch and do that before merge?

Reply:
sure, roger that